### PR TITLE
feat(dataset): add lakeview-nightlights-tornadoes-2024 dataset config

### PR DESCRIPTION
**Pull request by anayeaye**
## What
Add stac collection and ingestion config for `lakeview-nightlights-tornadoes-2024`

## Parent issue
https://github.com/NASA-IMPACT/veda-data/issues/187

## Downstream veda-config PR
https://github.com/NASA-IMPACT/veda-config/pull/420

## Checklist
- [ ] align title and description with veda-config
- [ ] staging config completed and verified
- [ ] published to staging
- [ ] transfer config and production metadata updated
- [ ] published to production

### CODE REVIEW

File: ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{
+    "collection": "lakeview-nightlights-tornadoes-2024",
+    "title": "Black Marble Night Lights– 2024 Spring Tornadoes Damage (Lakeview, OH)",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json:3]:
```suggestion
    "title": "Black Marble Night Lights (Lakeview OH Tornado Damage; Difference)",
```

File: ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    "description": "NASA Black Marble Night Lights shows the path of a tornado through Lakeview, Ohio on March 14th, 2024, revealing where power loss occurred due to this tornado strike on the city.",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json:4]:
```suggestion
    "description": "Black Marble night lights imagery of tornado damage at Lakeview, Ohio in the spring 2024 tornado season.",
```

File: ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            ],
+            "title": "Default COG Layer",
+            "description": "Cloud optimized default layer to display on map"
+        }
+    },
+    "renders": {
+        "dashboard": {
+            "resampling": "{nearest CHANGE ME}",
+            "bidx": [1],
+            "colormap_name": "{CHANGE ME}",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json:73]:
```suggestion
            "colormap_name": "bwr",
```

File: ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            "nodata": "{CHANGE ME}",
```
Comment (botanical) [inline ingestion-data/staging/dataset-config/lakeview-nightlights-tornadoes-2024.json:74]:
```suggestion
            "nodata": "-999",
```

**Review by botanical — CHANGES_REQUESTED** [review]
@smohiudd did you work on this dataset? I think you need to commit the files changes and add the prod dataset-config and transfer-config 😯

**Comment by smohiudd** [comment]
@botanical thanks for the catch. Just pushed the prod files now.

**Review by botanical — APPROVED** [review]

**Pull request merged**